### PR TITLE
CI: Build&Pack package with QUICSDK

### PR DESCRIPTION
**Pull request by lijinxia**

### CODE REVIEW

File: test/server.jenkinsfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ pipeline {
                 podTemplate(name: "pack-server-centos-"+env.GIT_BRANCH, label: "pack-server-centos-"+env.GIT_BRANCH,
                 volumes: [
                 persistentVolumeClaim(claimName: 'centos-package',mountPath: '/root/centos_package',readOnly: false),
+		persistentVolumeClaim(claimName: 'quicsdk',mountPath: '/root/quicsdk',readOnly: false),
```
Comment (starwarfan) [inline test/server.jenkinsfile:24]: Use space instead of tab and keep the indent same.

File: test/ubuntuserver.jenkinsfile [hunk 2]
```diff
@@ -20,7 +20,8 @@ pipeline {
             steps {
                 podTemplate(name: 'pack-server-ubuntu-'+env.GIT_BRANCH, label: 'pack-server-ubuntu-'+env.GIT_BRANCH,
                 volumes: [
-        	    persistentVolumeClaim(claimName: 'ubuntu-package',mountPath: '/root/ubuntu_package',readOnly: false),
+        	      persistentVolumeClaim(claimName: 'ubuntu-package',mountPath: '/root/ubuntu_package',readOnly: false),
```
Comment (starwarfan) [inline test/ubuntuserver.jenkinsfile:23]: Avoid mixing spaces and tabs for indent. Use spaces only.

File: test/server.jenkinsfile (hunk 1 above)
Comment (lijinxia) [reply test/server.jenkinsfile]: Done!

File: test/ubuntuserver.jenkinsfile (hunk 2 above)
Comment (lijinxia) [reply test/ubuntuserver.jenkinsfile]: Done!

File: test/ubuntuserver.jenkinsfile [hunk 3]
```diff
@@ -20,9 +20,10 @@ pipeline {
             steps {
                 podTemplate(name: 'pack-server-ubuntu-'+env.GIT_BRANCH, label: 'pack-server-ubuntu-'+env.GIT_BRANCH,
                 volumes: [
-        	    persistentVolumeClaim(claimName: 'ubuntu-package',mountPath: '/root/ubuntu_package',readOnly: false),
+                persistentVolumeClaim(claimName: 'ubuntu-package',mountPath: '/root/ubuntu_package',readOnly: false),
+                persistentVolumeClaim(claimName: 'quicsdk',mountPath: '/root/quicsdk',readOnly: false),
                 persistentVolumeClaim(claimName: 'webrtc88ubuntu',mountPath: '/root/owt-server/third_party/webrtc-m88',readOnly: false)
-        	    ],
+        	],
```
Comment (starwarfan) [inline test/ubuntuserver.jenkinsfile:26]: Here is still a tab before ']'. You can configure you editor to use space instead of tab.
Anyway, it's ok.

**Review by starwarfan — APPROVED** [review]

**Pull request merged**